addon/index.js

```diff
@@ -3,6 +3,8 @@ import Response from './response';
 import faker from './faker';
 import Model from './orm/model';
 import Serializer from './serializer';
+import ActiveModelSerializer from './serializers/active-model-serializer';
+import JSONAPISerializer from './serializers/json-api-serializer';
```

Comment:
Let's stick with `JSONAPISerializer` to match Ember Data, and if it's different anywhere else, let's change it.